src/org/omegat/util/StaticUtils.java

```diff
@@ -445,15 +443,14 @@ private static void replaceGlobs(StringBuilder haystack, String needle, String r
     }
 
     /**
-     * Download a file to memory
+     * Download a file to memory.
+     * @Deprecated
+     * This method is replaced to HttpConnectionUtils.getURL(url)
+     * No longer accept timeout.
```

Comment:
Was there a reason to get rid of the timeout parameter?

Reply:
Functions which use the function set a same timeout value so I remove it. Now there is a compat function so I'd like to revert to support timeout.

Reply:
Now accept timeout.